docs/sources/k6/next/using-k6-browser/recommended-practices/preventing-too-many-time-series-issue.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Preventing too many time series issue'
```

Comment:
```suggestion
title: 'Prevent too many time series error'
```

I think "error" sounds better, and it also matches the content below. What do you think?

Would also need to update the h1 heading.